examples/chat-room-reactions/javascript/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="src/styles.css" />
+    <script type="module" src="src/script.ts"></script>
+    <title>Chat room reactions Demo</title>
+  </head>
+  <body style="font-family: 'Inter';font-size: 22px;">
+    <div id="chat-room-reactions" class="container">
+      <div class="flex-1 p:2 sm:p-12 justify-between flex flex-col h-screen">
+        <div
+          id="messages"
+          class="w-96 flex flex-auto flex-col space-y-4 p-3 overflow-y-auto scrollbar-thumb-blue scrollbar-thumb-rounded scrollbar-track-blue-lighter scrollbar-w-2 scrolling-touch"
+        >
+        </div>
+        <div class="border-t-2 border-gray-200 px-4 pt-4 mb-2 sm:mb-0">
+          <form
+            class="flex"
+          >
+            <input
+               type="text"
+              disabled
+              placeholder="Say something"
```

Comment:
Should we have something along the lines of `Type a message...` as the placeholder (and add italics?)